tests/csrf_tests/tests.py

```diff
@@ -693,6 +693,19 @@ def test_process_response_get_token_used(self):
         ensure_csrf_cookie_view(req)
         self.assertTrue(req.session.get(CSRF_SESSION_KEY, False))
 
+    def test_session_modify(self):
+        """The session should not be modified if not necessary."""
+        req = self._get_GET_no_csrf_cookie_request()
+        self.mw.process_view(req, ensure_csrf_cookie_view, (), {})
+        resp = ensure_csrf_cookie_view(req)
+        self.mw.process_response(req, resp)
+        self.assertTrue(req.session.get(CSRF_SESSION_KEY, False))
```

Comment:
I think `assertIsNotNone(req.session.get(CSRF_SESSION_KEY))` would better express the existence assertion here.

Reply:
Well I've shamelessly copied the code in above function, but you're right.